DemoApp/Assets/Scripts/HomeScreen.cs

```diff
@@ -348,6 +371,17 @@ private void OnRewardedInterstitialAdRevenuePaidEvent(string adUnitId, MaxSdkBas
         string networkName = adInfo.NetworkName; // Display name of the network that showed the ad (e.g. "AdColony")
         string adUnitIdentifier = adInfo.AdUnitIdentifier; // The MAX Ad Unit ID
         string placement = adInfo.Placement; // The placement this ad's postbacks are tied to
+        
+        // initialise with AppLovin MAX source
+        AdjustAdRevenue adjustAdRevenue = new AdjustAdRevenue(AdjustConfig.AdjustAdRevenueSourceAppLovinMAX);
+        // set revenue and currency
+        adjustAdRevenue.setRevenue(revenue, "USD");
+        // optional parameters
+        adjustAdRevenue.setAdRevenueNetwork(networkName);
+        adjustAdRevenue.setAdRevenueUnit(adUnitIdentifier);
+        adjustAdRevenue.setAdRevenuePlacement(placement);
+        // track ad revenue
+        Adjust.trackAdRevenue(adjustAdRevenue);

```

Comment:
Unlike iOS or Android demos - since our callbacks are all in the same class, let's just abstract it into a function. Also  I'm going to delete the comments - it's pretty obvious what's happening and you don't have them on native either.
```suggestion
        AdjustAdRevenue adjustAdRevenue = new AdjustAdRevenue(AdjustConfig.AdjustAdRevenueSourceAppLovinMAX);
        adjustAdRevenue.setRevenue(revenue, "USD");
        adjustAdRevenue.setAdRevenueNetwork(networkName);
        adjustAdRevenue.setAdRevenueUnit(adUnitIdentifier);
        adjustAdRevenue.setAdRevenuePlacement(placement);
        
        Adjust.trackAdRevenue(adjustAdRevenue);

```